[2025-02-09T20:19:45.578+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T20:19:45.597+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:19:44.158290+00:00 [queued]>
[2025-02-09T20:19:45.612+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:19:44.158290+00:00 [queued]>
[2025-02-09T20:19:45.615+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T20:19:45.637+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Upload_data_to_s3> on 2025-02-09 20:19:44.158290+00:00
[2025-02-09T20:19:45.644+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=318) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T20:19:45.645+0000] {standard_task_runner.py:72} INFO - Started process 319 to run task
[2025-02-09T20:19:45.646+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_to_scrape_and_upload', 'Upload_data_to_s3', 'manual__2025-02-09T20:19:44.158290+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/scrape_upload_dag.py', '--cfg-path', '/tmp/tmpipc26kkw']
[2025-02-09T20:19:45.647+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask Upload_data_to_s3
[2025-02-09T20:19:45.703+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_to_scrape_and_upload.Upload_data_to_s3 manual__2025-02-09T20:19:44.158290+00:00 [running]> on host 6a5177197bb5
[2025-02-09T20:19:45.822+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_to_scrape_and_upload' AIRFLOW_CTX_TASK_ID='Upload_data_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T20:19:44.158290+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T20:19:44.158290+00:00'
[2025-02-09T20:19:45.824+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T20:20:01.574+0000] {logging_mixin.py:190} INFO - {'error': 'Parameter validation failed:\nUnknown parameter in input: "config", must be one of: ACL, Body, Bucket, CacheControl, ContentDisposition, ContentEncoding, ContentLanguage, ContentLength, ContentMD5, ContentType, ChecksumAlgorithm, ChecksumCRC32, ChecksumCRC32C, ChecksumSHA1, ChecksumSHA256, Expires, IfNoneMatch, GrantFullControl, GrantRead, GrantReadACP, GrantWriteACP, Key, Metadata, ServerSideEncryption, StorageClass, WebsiteRedirectLocation, SSECustomerAlgorithm, SSECustomerKey, SSECustomerKeyMD5, SSEKMSKeyId, SSEKMSEncryptionContext, BucketKeyEnabled, RequestPayer, Tagging, ObjectLockMode, ObjectLockRetainUntilDate, ObjectLockLegalHoldStatus, ExpectedBucketOwner'}
[2025-02-09T20:20:01.580+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-09T20:20:01.594+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T20:20:01.595+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_to_scrape_and_upload, task_id=Upload_data_to_s3, run_id=manual__2025-02-09T20:19:44.158290+00:00, execution_date=20250209T201944, start_date=20250209T201945, end_date=20250209T202001
[2025-02-09T20:20:01.635+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T20:20:01.653+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T20:20:01.657+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
